numpy/core/include/numpy/ndarraytypes.h

```diff
@@ -401,10 +401,10 @@ typedef int (PyArray_FillFunc)(void *, npy_intp, void *);
 typedef int (PyArray_SortFunc)(void *, npy_intp, void *);
 typedef int (PyArray_ArgSortFunc)(void *, npy_intp *, npy_intp, void *);
 typedef int (PyArray_PartitionFunc)(void *, npy_intp, npy_intp,
-                                    npy_intp *, npy_intp *,
+                                    npy_intp *, npy_intp *, npy_intp,
                                     void *);
 typedef int (PyArray_ArgPartitionFunc)(void *, npy_intp *, npy_intp, npy_intp,
-                                       npy_intp *, npy_intp *,
+                                       npy_intp *, npy_intp *, npy_intp,
                                        void *);
```

Comment:
@seberg is there any reason these typedefs are public API? If not we should move them out of the public headers ...

Reply:
Yes, they are public API, legacy user dtypes may use them.

Reply:
What would be the appropriate place to move it to?

Reply:
`npy_partition.h`?

Reply:
Done.